app/controllers/courses/age_range_controller.rb

```diff
@@ -43,6 +27,48 @@ def update
 
   private
 
+    def update_age_range_param
+      if valid_custom_age_range?
+        params[:course][:age_range_in_years] = "#{age_from_param}_to_#{age_to_param}"
+      elsif age_range_is_other?
```

Comment:
I don't think this `elsif` branch is required anymore as it is now dealt earlier on in the process via the `build_errors` method. I will investigate the next PR that deals with the 'errors' bug that @misaka spotted